### PR TITLE
Create Translation GraphQL Query

### LINKED ISSUE #15: Create getTranslationResults() GraphQL query on BE
Takes in a target language, source language, target method, and returns array of five result objects (and one object for target method) which each have:

link to page & location in docs
confidence/relevance rating
method/function name
code snippet
code sequence (example usage)
description

(Enhance later with user-submitted solutions)

Example query + response:

```
getTranslatedMethods(sourceLang: 'Ruby', targetLang: 'JS', methodName: 'split') {
  methods {
    name
    docsUrl (ruby: anchor tag for specific method, JS: page for that method)
    description
    callSequence
    snippet
    relevance
  }
}
```

```js
[
  {
    name: 'map',
    docsUrl: 'mdndocs.com/JS/map',
    callSequence: 'thing(thing1, thing2)',
    snippet: 'some code',
    description: 'iterates over array and returns array of return values',
    relevance: 0.91
  },
  {
    name: 'map',
    docsUrl: 'mdndocs.com/JS/map',
    description: 'iterates over array and returns array of return values',
    relevance: 0.65
  },
  {
    name: 'map',
    docsUrl: 'mdndocs.com/JS/map',
    description: 'iterates over array and returns array of return values',
    relevance: 0.99
  },
  {
    name: 'map',
    docsUrl: 'mdndocs.com/JS/map',
    description: 'iterates over array and returns array of return values',
    relevance: 0.43
  },
  {
    name: 'map',
    docsUrl: 'mdndocs.com/JS/map',
    description: 'iterates over array and returns array of return values',
    relevance: 0.21
  }
]
```

### PR DESCRIPTION
**Pull request by DanielEFrampton**
### Sidebar Checklist
​
- [x] Ran test suite
- [x] Request reviewers
- [x] Assign yourself and other contributors
- [x] Link to project
- [x] Added to Readme
​
### Issues Resolved
Resolves #15 
​
### Problem Addressed
Front-end had no GraphQL query that could return the search results needed for core functionality.​

### Solution Implemented
​Created GraphQL query `translations` which takes target language ID and given source method ID and returns top five methods in target language sorted by relevance rating in descending order.

### Screenshots (optional)
Trying to get Graphene to do what we want:
​

Realizing we'd spent ~2 hours trying to get CircleCI to have a database and then realizing the tests passed all along:

### Other Notes
We did a lot of...preliminary research...into issue #30, and will pick it up again another time. We're also ready to tackle setting up Pytest to have a test database when we have an extra day or five.

### CODE REVIEW

File: flaskr/app.py [hunk 1]
```diff
@@ -118,6 +114,19 @@ class Query(graphene.ObjectType):
     node = graphene.relay.Node.Field()
     all_methods = SQLAlchemyConnectionField(MethodObject)
     all_languages = SQLAlchemyConnectionField(LanguageObject)
+    translations = graphene.Field(lambda: graphene.List(MethodResultObject),
+                                  method_id=graphene.Int(required=True),
+                                  target_language_id=graphene.Int(required=True))
+
+    def resolve_translations(parent, info, method_id, target_language_id):
```
Comment (jobannon) [inline flaskr/app.py:121]: Interesting ... parent.  This is the first time that I have seen this in python.  Although I am sure it's common.  Getting used to using parent and self.

File: flaskr/app.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 from graphene_sqlalchemy import SQLAlchemyObjectType, SQLAlchemyConnectionField
 from flask_graphql import GraphQLView
 from flask_cors import CORS
+from sqlalchemy import desc
```
Comment (jobannon) [inline flaskr/app.py:10]: Wondering about the full meaning of this.  Is this just for the query?

File: README.md [hunk 3]
```diff
@@ -99,6 +99,86 @@ If you need to clear your database, run the following commands:
 (This is intended for current developers and will be edited before final docs are published)
```
Comment (jobannon) [inline README.md:99]: Thanks for updating the Readme!

File: flaskr/app.py (hunk 2 above)
Comment (mintona) [reply flaskr/app.py:10]: Yup. Desc needs to be specifically imported from sqlalchemy, you can't call it if you don't import it. Kind of like how you have to import certain math functions from modules to use them.

**Pull request merged**